packages/nimble-components/src/wafer-map/styles.ts

```diff
@@ -53,5 +61,17 @@ export const styles = css`
 
     .wafer-map-area {
         position: absolute;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+    }
+
+    .wafer-map-canvas {
+        display: inline-block;
+        width: inherit;
+        height: inherit;
```

Comment:
These are being immediately overridden by the following height: 100% and width: 100% definition and can be removed.

Reply:
applied